### DISCUSSION TITLE
Possible license issues

### DISCUSSION THREAD

**Discussion by mousethief** [Sakati84/3DPrintColorizer#38] [General]
I didn't notice until seeing the thingiverse link but there may be some license conflicts.
Yes, it's okay to offer two licenses, but you would need to list them both on both sites, and I'm guessing that isn't what is actually meant here. Also, the NC (non-commercial) license on thingiverse is in conflict with the GPL (and most other copyleft-type licenses with a 'fields of endeavour' clause), and may not mean what you think it means. I admit I was surprised when I actually read the CC license. Please see (https://creativecommons.org/faq/#does-my-use-violate-the-noncommercial-clause-of-the-licenses) and (https://wiki.creativecommons.org/wiki/NonCommercial_interpretation) - I was dismayed to learn that (for instance) my local maker space, which is a 501c3 non-profit, is still likely a commercial user under the CC.
Dual licensing can work, if that's what you actually want, but it may well be over-complicated for a project such as this.

Also,
I know that while it's technically possible to edit stl files in a useful way, that's usually not the way people design 3D printable items. And the GPL is a promise to supply source code, which is usually defined as the form used to create, and make and share edits.

And I do wish to contribute!
I hope this helps,
-- beata (the mouse thief)

**Comment by Sakati84** [comment]
Thanks for the hint regarding the license I will change the thingiverse one. I will provide some step files soon. I'm on holidays right now and can't access my PC right now so it will take 2-3 days.

**Reply by mousethief** [reply]
No worries, enjoy holidays.
